File: C++/Iniciante/1043.cpp
#include<iostream>
#include<iomanip>
#include<cmath>

using namespace std;
  
int main(){
    
    double A, B, C, S;
    
    cout << fixed << setprecision(1);
    cin >> A >> B >> C;
    
	if((A<(B+C)) && (B<(A+C)) && (C<(A+B))){
		S=A+B+C;
		cout << "Perimetro = " << S << endl;
	}else{
		S=((A+B)*C)/2;
		cout << "Area = " << S << endl;
	}
	
	return(0);
}

File: Python3/Iniciante/1043.py
# -*- encoding: utf-8 -*-

N = input()

A = float(N.split()[0])
B = float(N.split()[1])
C = float(N.split()[2])

if (A+B<=C or A+C<=B or B+C<=A):
    AREA = ((A+B)*C)/2
    print("Area = %.1f" % AREA)
else:
    PERIMETRO = A+B+C
    print("Perimetro = %.1f" % PERIMETRO)
